src/test/java/uk/nhs/tis/sync/job/PostFundingStatusSyncJobTest.java

```diff
@@ -122,15 +122,15 @@ public void testShouldPassWhenMultipleFundingsAllEndDatesYesterday() {
   }
 
   @Test
-  void testShouldPassWhenAPostHasMultiplePostFundings() {
+  void testShouldPassWhenAPostHasMultiplePostFundingsEndDateNotYesterday() {
```

Comment:
This test case doesn't cover the senarios below:
1. a post has 2 fundings, between which, 1 funding ends yesterday and the other ends 1 month ago.
2. a post has 2 fundings, between which, 1 funding ends yesterday and the other ends in 1 month.
3. a post has 2 fundings, between which, 1 funding ends yesterday and the other has an empty enddate.